### DISCUSSION TITLE
mj-columno not breakpoint

### DISCUSSION THREAD

**Discussion by LuisMejiaPoot** [mjmlio/mjml#2604] [Q&A]
what would be the best way to breakpoint the fifth mj-column. 

when it has more than 5 mj-columns it does not make the breakpoint and does not put the sixth mj-column down.

This issue only happens in outlook

**Result below:**

![image](https://user-images.githubusercontent.com/43623174/206581336-ae4dca68-a45a-417d-81b2-ab61b2afe61e.png)

**Comment by iRyusa** [comment]
You can't have more than 100% of content inside a section. Break it into a
new section as outlook doesn't know how to wrap columns as a browser would
